game/scripts/vscripts/abilities/swiper/boss_swiper_swipe.lua

```diff
@@ -0,0 +1,80 @@
+function DebugRange(caster, range, ability)
```

Comment:
avoid global function, this could conflict with other functions if we starts making them global.
either move to a helper class on components or put it on the ability/modifier class